admin/module/catalog-admin-module-sources/src/main/webapp/js/view/Source.view.js

```diff
@@ -78,6 +78,9 @@ function (ich,Marionette,_,$,Q,ModalSource,EmptyView,Service,Status,wreqr,Utils,
             }
             data.available = this.model.get('available');
             data.name = this.model.get('name');
+            if (!data.name && data.currentConfiguration && data.currentConfiguration.id) {
```

Comment:
I don't agree at all that overriding serializeData is a code smell. That function exists specifically for the case where you want to present information to the template that you do not want to store inside your model. That being said, I agree with Andrew.